libs/coin-evm/docs/buildOptimisticOperation.md

```diff
@@ -0,0 +1,19 @@
+# `buildOptimisticOperation.ts`
```

Comment:
Duplicated in the code

Reply:
I think this documentation and the JSDoc should have slightly different purposes: the JSDoc should be a relatively succinct description while this doc should go into details as much as necessary. 

I'll advise to keep the 2 separated and improve with more details if necessary this markdown rather than removing it, which will also go in the direction I was mentioning before which is having a complete documentation right away, without having to go look into the code to find something not written here.